utils/wesnoth-map-diff/src/images.ts

```diff
@@ -22,6 +22,9 @@ const getDictTerrainType2ImagesPath = async () => {
       }
```

Comment:
> idk if this is even actually valid TypeScript, much less whether it will actually work or not

Can you follow the package readme? It explains how to run it.
Let me know if you need some help on it

Reply:
I did when I first created this branch, but that was a few months ago and I forgot about it... (it's also how I ended up with the npm dependency that you told me to remove in [a24458d](https://github.com/wesnoth/wesnoth/pull/7452/commits/a24458d5d204f6effabffbe3d4fb0958e2123af0))

Reply:
ok I just tried doing `npm run build:dev` and this was the output it printed:
```
npm run build:dev
npm WARN cli npm v9.6.0 does not support Node.js v15.14.0. You should probably upgrade to a
npm WARN cli newer version of node as we can't make any promises that npm will work with this
npm WARN cli version. This version of npm supports the following node versions: `^14.17.0 ||
npm WARN cli ^16.13.0 || >=18.0.0`. You can find the latest version at https://nodejs.org/.

> wesnoth-map-diff@0.0.1 build:dev
> tsc -p tsconfig-dev.json

src/images.ts:40:7 - error TS2552: Cannot find name 'image'. Did you mean 'Image'?

40       image = await Jimp.read(`${imageBasepath}/${imageName}.png`)
         ~~~~~

  node_modules/typescript/lib/lib.dom.d.ts:17148:13
    17148 declare var Image: {
                      ~~~~~
    'Image' is declared here.

src/images.ts:41:12 - error TS2552: Cannot find name 'image'. Did you mean 'Image'?

41       if (!image) {
              ~~~~~

  node_modules/typescript/lib/lib.dom.d.ts:17148:13
    17148 declare var Image: {
                      ~~~~~
    'Image' is declared here.

src/images.ts:42:9 - error TS2552: Cannot find name 'image'. Did you mean 'Image'?

42         image = await Jimp.read(`../../../images/${imageName}.png`)
           ~~~~~

  node_modules/typescript/lib/lib.dom.d.ts:17148:13
    17148 declare var Image: {
                      ~~~~~
    'Image' is declared here.

src/images.ts:44:28 - error TS2552: Cannot find name 'image'. Did you mean 'Image'?

44       return [terrainType, image]
                              ~~~~~

  node_modules/typescript/lib/lib.dom.d.ts:17148:13
    17148 declare var Image: {
                      ~~~~~
    'Image' is declared here.


Found 4 errors.

```
...is that because I removed the `const` from `image`? I thought that would be necessary to make a second attempt at assigning to it...

Reply:
yeah, if I put the `const` back, I get:
```
> tsc -p tsconfig-dev.json

src/images.ts:42:9 - error TS2588: Cannot assign to 'image' because it is a constant.

42         image = await Jimp.read(`../../../images/${imageName}.png`)
           ~~~~~


Found 1 error.
```

Reply:
> .is that because I removed the const from image? I thought that would be necessary to make a second attempt at assigning to it...

You always need to declare a variable before using it.
If you need to have a mutable variable, use `let`.
```diff
- const image = await Jimp.read(`${imageBasepath}/${imageName}.png`)
+ let image = await Jimp.read(`${imageBasepath}/${imageName}.png`)
```